src/app/technical-record/adr/certificate/edit/certificate-edit.component.spec.ts

```diff
@@ -19,7 +25,11 @@ describe('CertificateEditComponent', () => {
         FormGroupDirective,
         {
           provide: FormGroupDirective,
-          useValue: TESTING_UTILS.mockFormGroupDirective()
+          useValue: TESTING_UTILS.mockFormGroupDirective({
+            techRecord: new FormGroup({
+              adrDetails: new FormGroup({})
+            }) as AbstractControl
+          })
```

Comment:
We are writing the same mock in almost every test. Can this be centralised?

Reply:
nope and we should not. Every test should be localised and focused and so should be every component